jekyll-redirect-from.gemspec

```diff
@@ -24,7 +24,8 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = ">= 2.3.0"
 
-  spec.add_runtime_dependency "jekyll", "~> 3.3"
+  spec.add_runtime_dependency "jekyll", ">= 3.3", "< 5.0"
```

Comment:
Given we don't backport security updates to former versions, why dont we ask for `>= 3.7` here?
Most of other plugins do anyway.

Reply:
That decision is upto the user.
The plugin should not force the user to a newer version as long as it doesn't use a functionality from  a newer version.